### ISSUE TITLE
JSON Scheme version support

### ISSUE THREAD

**Issue by subha84** [guillotinaweb/ngx-schema-form#238]
does 'ngx-schema-form' supports latest json schema draft 06/07 ?

**Comment by subha84** [comment]
No, But I am curious to know "if/then/else" schema syntax is supported by this tool ?
I am in process of evaluating many JSON schema to form generator and none of them supports beyond 
draft 3/4. I am hoping this must support as it is pretty latest.

**Comment by ebrehault** [comment]
The demo is fixed, thank you for reporting :) http://makinacorpus.github.io/ngx-schema-form/dist/ngx-schema-form/

We do manage conditional fields (see https://github.com/makinacorpus/ngx-schema-form#conditional-fields). The syntax is not the one from the specs, but that something which can be easily fxied on the fly when you pass your schema to the component.
